optimade/server/mappers/entries.py

```diff
@@ -1,11 +1,12 @@
-from typing import Tuple
+from typing import Tuple, Optional
 from optimade.server.config import CONFIG
 
 __all__ = ("BaseResourceMapper",)
 
 
 class BaseResourceMapper:
-    """ Generic Resource Mapper that defines and performs the mapping
+    """
```

Comment:
Maybe not. Maybe only for those places where the docs don't render properly.

Reply:
Yeah. So for exactly this doc string, it is indeed necessary to have a line break, otherwise the formatting doesn't work.
There's an issue with class doc strings.
However, for all other doc strings there doesn't seem to be issues.